frontend/src/types/interfaces.d.ts

```diff
@@ -0,0 +1,68 @@
+export interface User {
+  id: number;
+  first_name: string;
+  last_name: string;
+  email: string;
+  password: string;
+  phone_number: string;
+  user_summary: string;
+}
+
+export interface Friend {
+  id_1: number; 
+  id_2: number;
+ }
+
+export interface Blocked {
+  user_id: number;
+  blocked: number;
+}

```

Comment:
I don't think we have a way to block users in our test plans lol. We can keep it in for the time being, but maybe if we never implement it, this can probably be commented out

Reply:
Yeah this will also most likely be an artifact that gets either removed once we have the API or just not implemented.